packages/engine-server/src/markdown/remark/utils.ts

```diff
@@ -1292,13 +1294,13 @@ export class RemarkUtils {
       return [];
     }
     const nodesToExtract = nextHeaderIndex
-      // @ts-ignore
-      ? (tree.children as Node[]).splice(
+      ? // @ts-ignore
+        (tree.children as Node[]).splice(
           foundHeaderIndex!,
           nextHeaderIndex! - foundHeaderIndex!
         )
-      // @ts-ignore
-      : (tree.children as Node[]).splice(foundHeaderIndex!);
+      : // @ts-ignore
```

Comment:
Could get rid of `@ts-ignore` and `as Node[]` by changing the code a bit:
```ts
    if (!headerFound || !RemarkUtils.isParent(tree)) {
      return [];
    }
    const nodesToExtract = nextHeaderIndex
      ? tree.children.splice(
          foundHeaderIndex!,
          nextHeaderIndex! - foundHeaderIndex!
        )
      : tree.children.splice(foundHeaderIndex!);
    return nodesToExtract;
```

Reply:
5be00ce